config.yaml.defaults

```diff
@@ -60,6 +60,18 @@ ports:
     websocket_path_in: 'ipc://run/message_flow_in'
     websocket_path_out: 'ipc://run/message_flow_out'
 
+
+external_logging:
+    enabled: False
+    # get an account at https://papertrailapp.com
+    service_name: papertrail
+    # change url to the correct subdomain for your account
+    # and change the port as well
+    url: logX.papertrailapp.com
+    port: XX
+    # which log files, if any do you not want to send over to the 3rd party?
+    excluded_log_files: []
```

Comment:
```suggestion
    services:
        - papertrail:
            # change url to the correct subdomain for your account
            # and change the port as well
            url: logX.papertrailapp.com
            port: XX
            # which log files, if any do you not want to send over to the 3rd party?
            excluded_log_files: []
```

Reply:
Suggesting the above because: 

1. Other services may have different config parameters beyond url, port, excluded log files
2. We may want to support logging to multiple services

Reply:
Done.